docs/built-in-models/neural-networks.md

```diff
@@ -17,17 +17,14 @@ While `NeuralNetClassifier` and `NeuralNetClassifier` provide scikit-learn-like
 Good quantization parameter values are critical to make models [respect FHE constraints](../getting-started/concepts.md#model-accuracy-considerations-under-fhe-constraints). Weights and activations should be quantized to low precision (e.g., 2-4 bits). The sparsity of the network can be tuned [as described below](neural-networks.md#overflow-errors) to avoid accumulator overflow.
 {% endhint %}
 
+{% hint style="warning" %}
+Using `nn.ReLU` as the activation function benefits from an optimization where quantization uses powers-of-two scales. This results in much faster inference times in FHE, thanks to a TFHE primitive that performs fast division by powers of two.
```

Comment:
do we have a documentation to link about it ?

Reply:
added one